src/sql/workbench/parts/notebook/cellViews/outputArea.css

```diff
@@ -14,4 +14,9 @@ output-area-component .notebook-output {
 
 .output-userselect.actionselect {
 	user-select: text;
+}
+
+.output-userselect pre{
+	white-space: pre-wrap;
```

Comment:
nitpick: indentation here looks wrong. One is more indented than the other. Are both 1 tab?